src/actions/index.js

```diff
@@ -118,6 +119,11 @@ export const createNewProjectFinish = () => ({
   type: CREATE_NEW_PROJECT_FINISH,
 });
 
+export const setDefaultProjectPath = (path: string) => ({
+  type: SET_DEFAULT_PROJECT_PATH,
+  path,
+});
```

Comment:
Could you please rename this action to `changeDefaultProjectHome`?

- I think `change` is better than `set` because it's clear that the user is changing something, and it's not just setting a property on a model. It's user-directed.
- `homePath` instead of `path` because it's setting the parent "home" directory, not the project directory itself.